syncd/syncd_applyview.cpp

```diff
@@ -1203,6 +1796,7 @@ sai_status_t applyViewTransition(
     // and then we will need second pass on all objects to see
     // if we have the same attributes
 
+    //return SAI_STATUS_SUCCESS;
     return SAI_STATUS_NOT_IMPLEMENTED;
```

Comment:
when i develop i change it time to time since version with success produces more checks and produces a lot of noise, but time to time i want to see if it actually works to produce noise logs and not crashes, and forgot to remove, in final version in this way should return success